core/object/object.h

```diff
@@ -300,158 +301,166 @@ struct ObjectNativeExtension {
  * much alone defines the object model.
  */
 
-#define REVERSE_GET_PROPERTY_LIST                                  \
-public:                                                            \
-	_FORCE_INLINE_ bool _is_gpl_reversed() const { return true; }; \
-                                                                   \
+#define REVERSE_GET_PROPERTY_LIST                  \
+public:                                            \
```

Comment:
Huge diff due to clang-format 14 bug, needs to be fixed to be like before style wise.

Reply:
Resolved in newer pr.